{"TimeStamp":"1738710024", "Location":"/var/log/syslog", "Content":"Feb  4 23:00:23 debian systemd[19609]: Listening on GnuPG cryptographic agent and passphrase cache (access for web browsers).\nFeb  4 23:00:23 debian systemd[19609]: Listening on GnuPG cryptographic agent and passphrase cache (restricted).\nFeb  4 23:00:23 debian systemd[19609]: Listening on GnuPG cryptographic agent (ssh-agent emulation).\nFeb  4 23:00:23 debian systemd[19609]: Listening on GnuPG cryptographic agent and passphrase cache.\nFeb  4 23:00:23 debian systemd[19609]: Reached target Sockets.\nFeb  4 23:00:23 debian systemd[19609]: Reached target Basic System.\nFeb  4 23:00:23 debian systemd[1]: Started User Manager for UID 1001.\nFeb  4 23:00:23 debian systemd[1]: Started Session 120 of user blankcanvas.\nFeb  4 23:00:23 debian systemd[19609]: Reached target Main User Target.\nFeb  4 23:00:23 debian systemd[19609]: Startup finished in 49ms.\nFeb  4 23:00:23 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:23] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:23 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:23] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:23 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:23] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:23 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:23] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:23 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:23] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:23 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:23] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:23 debian systemd[1]: session-120.scope: Succeeded.\nFeb  4 23:00:23 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:00:23] [Ansible] saving config specified in env\nFeb  4 23:00:23 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:00:23] [Ansible] saving config and playbooks from home\nFeb  4 23:00:23 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:00:23] [Ansible] saving config from ~ and /etc\nFeb  4 23:00:23 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:00:23] [Ansible] finished reading. looping\nFeb  4 23:00:23 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:00:23] [Ansible] Saving playbooks from /etc/ansible\nFeb  4 23:00:23 debian discern-ansible-sorcerer[17057]: #033[31mERRO#033[0m[23:00:23] [Ansible] error reading /etc/ansible: stat /etc/ansible: no such file or directory\nFeb  4 23:00:24 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:24] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710024", "Location":"/var/log/auth.log", "Content":"Feb  4 23:00:23 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:23 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nFeb  4 23:00:23 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:23 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:23 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nFeb  4 23:00:23 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:23 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:23 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nFeb  4 23:00:23 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:23 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:23 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.97.trace\nFeb  4 23:00:23 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:23 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:23 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.97.trace\nFeb  4 23:00:23 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:23 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:23 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/rm -rf /var/log/discern/data/bash-data.txt /var/log/discern/data/cpu-load-data.txt /var/log/discern/data/file-data.txt /var/log/discern/data/logs-data.txt /var/log/discern/data/network-data.txt /var/log/discern/data/proc-cpu-data.txt /var/log/discern/data/proc-mem-data.txt /var/log/discern/data/proc-new-data.txt\nFeb  4 23:00:23 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:23 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:23 debian sshd[19627]: Received disconnect from 192.168.254.2 port 38220:11: disconnected by user\nFeb  4 23:00:23 debian sshd[19627]: Disconnected from user blankcanvas 192.168.254.2 port 38220\nFeb  4 23:00:23 debian sshd[19606]: pam_unix(sshd:session): session closed for user blankcanvas\nFeb  4 23:00:23 debian systemd-logind[390]: Session 120 logged out. Waiting for processes to exit.\nFeb  4 23:00:23 debian systemd-logind[390]: Removed session 120.\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710024", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:00:23 debian systemd[19609]: Listening on GnuPG cryptographic agent (ssh-agent emulation).\nFeb  4 23:00:23 debian systemd[19609]: Listening on GnuPG cryptographic agent and passphrase cache.\nFeb  4 23:00:23 debian systemd[19609]: Reached target Sockets.\nFeb  4 23:00:23 debian systemd[19609]: Reached target Basic System.\nFeb  4 23:00:23 debian systemd[1]: Started User Manager for UID 1001.\nFeb  4 23:00:23 debian systemd[1]: Started Session 120 of user blankcanvas.\nFeb  4 23:00:23 debian systemd[19609]: Reached target Main User Target.\nFeb  4 23:00:23 debian systemd[19609]: Startup finished in 49ms.\nFeb  4 23:00:23 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:23] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:23 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:23] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:23 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:23] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:23 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:23] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:23 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:23] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:23 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:23] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:23 debian systemd[1]: session-120.scope: Succeeded.\nFeb  4 23:00:23 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:00:23] [Ansible] saving config specified in env\nFeb  4 23:00:23 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:00:23] [Ansible] saving config and playbooks from home\nFeb  4 23:00:23 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:00:23] [Ansible] saving config from ~ and /etc\nFeb  4 23:00:23 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:00:23] [Ansible] finished reading. looping\nFeb  4 23:00:23 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:00:23] [Ansible] Saving playbooks from /etc/ansible\nFeb  4 23:00:23 debian discern-ansible-sorcerer[17057]: #033[31mERRO#033[0m[23:00:23] [Ansible] error reading /etc/ansible: stat /etc/ansible: no such file or directory\nFeb  4 23:00:24 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:24] [Logs] /var/log/dmesg not found. trying to capture from cli\nFeb  4 23:00:24 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:24] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:24 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:24] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710029", "Location":"/var/log/syslog", "Content":"Feb  4 23:00:25 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:25] [Proc] error saving proc-new info for pid 19610: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:25 debian discern-proc-sorcerer[18558]: #033[36mINFO#033[0m[23:00:25] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:00:25 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:00:25] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:25 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:25] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:25 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 217.\nFeb  4 23:00:25 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:25 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:25 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:26 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:26] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-cpu info for pid 18558: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-cpu info for pid 19609: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-mem info for pid 17257: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-mem info for pid 17232: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-mem info for pid 18558: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:29 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 218.\nFeb  4 23:00:29 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:29 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:29 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:29 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:29] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710029", "Location":"/var/log/auth.log", "Content":"Feb  4 23:00:29 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/open.bt\nFeb  4 23:00:29 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execveat.bt\nFeb  4 23:00:29 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execve.bt\nFeb  4 23:00:29 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat2.bt\nFeb  4 23:00:29 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvfrom.bt\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:29 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat.bt\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:29 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:29 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710029", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:00:25 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:00:25] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:25 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:25] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:25 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 217.\nFeb  4 23:00:25 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:25 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:25 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:26 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:26] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-cpu info for pid 18558: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-cpu info for pid 19609: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-mem info for pid 17257: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-mem info for pid 17232: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:27 debian discern-proc-sorcerer[18558]: #033[31mERRO#033[0m[23:00:27] [Proc] error saving proc-mem info for pid 18558: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:29 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 218.\nFeb  4 23:00:29 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:29 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:29 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:29 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:29] [Logs] /var/log/dmesg not found. trying to capture from cli\nFeb  4 23:00:29 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:29] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:29 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:29] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710034", "Location":"/var/log/syslog", "Content":"Feb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-56.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-57.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-58.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-59.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-6.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-60.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-61.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-62.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-63.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-64.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-65.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-66.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-67.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-68.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-69.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-7.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-70.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-71.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-72.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-73.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-74.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-75.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-76.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:34] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710034", "Location":"/var/log/auth.log", "Content":"Feb  4 23:00:31 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:31 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nFeb  4 23:00:31 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:31 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:31 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nFeb  4 23:00:31 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:31 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:31 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nFeb  4 23:00:31 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:31 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:31 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.98.trace\nFeb  4 23:00:31 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:31 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:31 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.98.trace\nFeb  4 23:00:31 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:31 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:31 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/systemctl restart discern-proc\nFeb  4 23:00:31 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:31 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:31 debian sshd[19712]: Received disconnect from 192.168.254.2 port 45440:11: disconnected by user\nFeb  4 23:00:31 debian sshd[19712]: Disconnected from user blankcanvas 192.168.254.2 port 45440\nFeb  4 23:00:31 debian sshd[19706]: pam_unix(sshd:session): session closed for user blankcanvas\nFeb  4 23:00:31 debian systemd-logind[390]: Session 122 logged out. Waiting for processes to exit.\nFeb  4 23:00:31 debian systemd-logind[390]: Removed session 122.\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710034", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-58.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-59.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-6.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-60.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-61.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-62.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:33 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:33] [Bash] Error saving cache-blankcanvas-63.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-64.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-65.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-66.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-67.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-68.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-69.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-7.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-70.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-71.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-72.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-73.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-74.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-75.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-bash-sorcerer[18364]: #033[31mERRO#033[0m[23:00:34] [Bash] Error saving cache-blankcanvas-76.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nFeb  4 23:00:34 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:34] [Logs] /var/log/dmesg not found. trying to capture from cli\nFeb  4 23:00:34 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:34] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:34 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:34] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710039", "Location":"/var/log/syslog", "Content":"Feb  4 23:00:35 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:35 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:35 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:35] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:35 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[36mINFO#033[0m[23:00:36] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:00:36 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:36] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-cpu info for pid 19746: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-cpu info for pid 17232: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-mem info for pid 17257: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-mem info for pid 17232: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-mem info for pid 19746: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:37 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:37] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:37 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:00:37] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:38 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:38] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:38 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 221.\nFeb  4 23:00:38 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:38 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:38 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:39 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:39] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710039", "Location":"/var/log/auth.log", "Content":"Feb  4 23:00:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:35 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710039", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:00:35 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:35] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:35 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[36mINFO#033[0m[23:00:36] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:00:36 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:36] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-cpu info for pid 19746: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-cpu info for pid 17232: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-mem info for pid 17257: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-mem info for pid 17232: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:36 debian discern-proc-sorcerer[19746]: #033[31mERRO#033[0m[23:00:36] [Proc] error saving proc-mem info for pid 19746: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:37 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:37] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:37 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:00:37] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:38 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:38] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:38 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 221.\nFeb  4 23:00:38 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:38 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:38 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:39 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:39] [Logs] /var/log/dmesg not found. trying to capture from cli\nFeb  4 23:00:39 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:39] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:39 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:39] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710044", "Location":"/var/log/syslog", "Content":"Feb  4 23:00:43 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:43] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian systemd[1]: Started Session 128 of user blankcanvas.\nFeb  4 23:00:43 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:43] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:43] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:43] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:43] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:43] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:43] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian systemd[1]: session-128.scope: Succeeded.\nFeb  4 23:00:43 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:00:43] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:43] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:43] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian systemd[1]: Started Session 129 of user blankcanvas.\nFeb  4 23:00:44 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:44] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:44] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:44] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:44] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:44] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:44] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian systemd[1]: session-129.scope: Succeeded.\nFeb  4 23:00:44 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:44] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:44] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian systemd[1]: Started Session 130 of user blankcanvas.\nFeb  4 23:00:44 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:44] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710044", "Location":"/var/log/auth.log", "Content":"Feb  4 23:00:44 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:44 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nFeb  4 23:00:44 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:44 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:44 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nFeb  4 23:00:44 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:44 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:44 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.105.trace\nFeb  4 23:00:44 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:44 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:44 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.105.trace\nFeb  4 23:00:44 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:44 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:44 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cp /etc/bind/named.conf.options-no-dnssec /etc/bind/named.conf.options\nFeb  4 23:00:44 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:00:44 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:44 debian sshd[20180]: Received disconnect from 192.168.254.2 port 48768:11: disconnected by user\nFeb  4 23:00:44 debian sshd[20180]: Disconnected from user blankcanvas 192.168.254.2 port 48768\nFeb  4 23:00:44 debian sshd[20174]: pam_unix(sshd:session): session closed for user blankcanvas\nFeb  4 23:00:44 debian systemd-logind[390]: Session 129 logged out. Waiting for processes to exit.\nFeb  4 23:00:44 debian systemd-logind[390]: Removed session 129.\nFeb  4 23:00:44 debian sshd[20215]: Accepted publickey for blankcanvas from 192.168.254.2 port 48778 ssh2: RSA SHA256:3I/WEXJMnChkvSdfEeoZ6YTUz2OrccqDruFNY41qR5o\nFeb  4 23:00:44 debian sshd[20215]: pam_unix(sshd:session): session opened for user blankcanvas(uid=1001) by (uid=0)\nFeb  4 23:00:44 debian systemd-logind[390]: New session 130 of user blankcanvas.\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710044", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:00:43 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:43] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:43] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:43] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:43] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:43] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:43] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian systemd[1]: session-128.scope: Succeeded.\nFeb  4 23:00:43 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:00:43] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:43] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:43] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:43 debian systemd[1]: Started Session 129 of user blankcanvas.\nFeb  4 23:00:44 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:44] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:44] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:44] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:44] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:00:44] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:44] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian systemd[1]: session-129.scope: Succeeded.\nFeb  4 23:00:44 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:44] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:44] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian systemd[1]: Started Session 130 of user blankcanvas.\nFeb  4 23:00:44 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:44] [Logs] /var/log/dmesg not found. trying to capture from cli\nFeb  4 23:00:44 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:44] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:44 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:44] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710049", "Location":"/var/log/syslog", "Content":"Feb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:2d::d#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:503:c27::2:30#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:9f::42#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:503:ba3e::2:30#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:a8::e#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:2f::f#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:1::53#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:7fd::1#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:12::d0d#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:2::c#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:7fe::53#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2801:1b8:10::b#53\nFeb  4 23:00:48 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:48 debian named[20369]: timed out resolving './DNSKEY/IN': 10.1.2.3#53\nFeb  4 23:00:48 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:2d::d#53\nFeb  4 23:00:48 debian named[20369]: managed-keys-zone: Initializing automatic trust anchor management for zone '.'; DNSKEY ID 20326 is now trusted, waiving the normal 30-day waiting period.\nFeb  4 23:00:48 debian named[20369]: managed-keys-zone: Initializing automatic trust anchor management for zone '.'; DNSKEY ID 38696 is now trusted, waiving the normal 30-day waiting period.\nFeb  4 23:00:48 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:48 debian named[20369]: resolver priming query complete\nFeb  4 23:00:48 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 224.\nFeb  4 23:00:48 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:48 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:48 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:49 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:49] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710049", "Location":"/var/log/auth.log", "Content":"Feb  4 23:00:48 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvmsg.bt\nFeb  4 23:00:48 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:48 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710049", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:9f::42#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:503:ba3e::2:30#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:a8::e#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:2f::f#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:1::53#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:7fd::1#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:12::d0d#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:2::c#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:7fe::53#53\nFeb  4 23:00:47 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2801:1b8:10::b#53\nFeb  4 23:00:48 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:48 debian named[20369]: timed out resolving './DNSKEY/IN': 10.1.2.3#53\nFeb  4 23:00:48 debian named[20369]: network unreachable resolving './DNSKEY/IN': 2001:500:2d::d#53\nFeb  4 23:00:48 debian named[20369]: managed-keys-zone: Initializing automatic trust anchor management for zone '.'; DNSKEY ID 20326 is now trusted, waiving the normal 30-day waiting period.\nFeb  4 23:00:48 debian named[20369]: managed-keys-zone: Initializing automatic trust anchor management for zone '.'; DNSKEY ID 38696 is now trusted, waiving the normal 30-day waiting period.\nFeb  4 23:00:48 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:48 debian named[20369]: resolver priming query complete\nFeb  4 23:00:48 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 224.\nFeb  4 23:00:48 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:48 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:48 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:49 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:49] [Logs] /var/log/dmesg not found. trying to capture from cli\nFeb  4 23:00:49 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:49] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:49 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:49] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710054", "Location":"/var/log/syslog", "Content":"Feb  4 23:00:49 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:49] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:49 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:49] [Logs] error saving /var/log/daemon.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:49 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:49] [Logs] Sending has completed\nFeb  4 23:00:49 debian systemd[1]: discern-proc.service: Scheduled restart job, restart counter is at 1.\nFeb  4 23:00:49 debian systemd[1]: Stopped \"The discern process info recording service\".\nFeb  4 23:00:49 debian systemd[1]: Started \"The discern process info recording service\".\nFeb  4 23:00:49 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:00:49] [Proc] hello world from Proc scraper\nFeb  4 23:00:49 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:00:49] [Proc] connected to fusion core\nFeb  4 23:00:49 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:00:49] [Proc] made intermediate data file folder\nFeb  4 23:00:49 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:00:49] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:00:49 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:00:49] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:51 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:51] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:51 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 225.\nFeb  4 23:00:51 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:51 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:51 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:54 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:54] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710054", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:00:49 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:49] [Logs] Sending has completed\nFeb  4 23:00:49 debian systemd[1]: discern-proc.service: Scheduled restart job, restart counter is at 1.\nFeb  4 23:00:49 debian systemd[1]: Stopped \"The discern process info recording service\".\nFeb  4 23:00:49 debian systemd[1]: Started \"The discern process info recording service\".\nFeb  4 23:00:49 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:00:49] [Proc] hello world from Proc scraper\nFeb  4 23:00:49 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:00:49] [Proc] connected to fusion core\nFeb  4 23:00:49 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:00:49] [Proc] made intermediate data file folder\nFeb  4 23:00:49 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:00:49] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:00:49 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:00:49] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:51 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:51] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:51 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 225.\nFeb  4 23:00:51 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:51 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:51 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:54 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:54] [Logs] /var/log/dmesg not found. trying to capture from cli\nFeb  4 23:00:54 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:54] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710059", "Location":"/var/log/syslog", "Content":"Feb  4 23:00:56 debian systemd[19609]: Closed GnuPG cryptographic agent and passphrase cache.\nFeb  4 23:00:56 debian systemd[19609]: Removed slice User Application Slice.\nFeb  4 23:00:56 debian systemd[19609]: Reached target Shutdown.\nFeb  4 23:00:56 debian systemd[19609]: systemd-exit.service: Succeeded.\nFeb  4 23:00:56 debian systemd[19609]: Finished Exit the Session.\nFeb  4 23:00:56 debian systemd[19609]: Reached target Exit the Session.\nFeb  4 23:00:56 debian systemd[1]: user@1001.service: Succeeded.\nFeb  4 23:00:56 debian systemd[1]: Stopped User Manager for UID 1001.\nFeb  4 23:00:56 debian systemd[1]: Stopping User Runtime Directory /run/user/1001...\nFeb  4 23:00:56 debian systemd[1]: run-user-1001.mount: Succeeded.\nFeb  4 23:00:56 debian systemd[1]: user-runtime-dir@1001.service: Succeeded.\nFeb  4 23:00:56 debian systemd[1]: Stopped User Runtime Directory /run/user/1001.\nFeb  4 23:00:56 debian systemd[1]: Removed slice User Slice of UID 1001.\nFeb  4 23:00:56 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:56] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:58 debian named[20369]: timed out resolving 'com/DS/IN': 10.1.2.3#53\nFeb  4 23:00:58 debian named[20369]: network unreachable resolving 'com/DS/IN': 2001:500:a8::e#53\nFeb  4 23:00:58 debian named[20369]: network unreachable resolving 'com/DS/IN': 2001:500:2f::f#53\nFeb  4 23:00:58 debian named[20369]: network unreachable resolving 'com/DS/IN': 2001:7fd::1#53\nFeb  4 23:00:58 debian named[20369]: network unreachable resolving 'com/DS/IN': 2001:503:ba3e::2:30#53\nFeb  4 23:00:58 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 227.\nFeb  4 23:00:58 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:58 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:58 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:59 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:59] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710059", "Location":"/var/log/auth.log", "Content":"Feb  4 23:00:55 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvfrom.bt\nFeb  4 23:00:55 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvmmsg.bt\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:55 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nFeb  4 23:00:55 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat2.bt\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:55 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:55 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:00:56 debian systemd: pam_unix(systemd-user:session): session closed for user blankcanvas\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710059", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:00:56 debian systemd[19609]: Reached target Shutdown.\nFeb  4 23:00:56 debian systemd[19609]: systemd-exit.service: Succeeded.\nFeb  4 23:00:56 debian systemd[19609]: Finished Exit the Session.\nFeb  4 23:00:56 debian systemd[19609]: Reached target Exit the Session.\nFeb  4 23:00:56 debian systemd[1]: user@1001.service: Succeeded.\nFeb  4 23:00:56 debian systemd[1]: Stopped User Manager for UID 1001.\nFeb  4 23:00:56 debian systemd[1]: Stopping User Runtime Directory /run/user/1001...\nFeb  4 23:00:56 debian systemd[1]: run-user-1001.mount: Succeeded.\nFeb  4 23:00:56 debian systemd[1]: user-runtime-dir@1001.service: Succeeded.\nFeb  4 23:00:56 debian systemd[1]: Stopped User Runtime Directory /run/user/1001.\nFeb  4 23:00:56 debian systemd[1]: Removed slice User Slice of UID 1001.\nFeb  4 23:00:56 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:00:56] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:58 debian named[20369]: timed out resolving 'com/DS/IN': 10.1.2.3#53\nFeb  4 23:00:58 debian named[20369]: network unreachable resolving 'com/DS/IN': 2001:500:a8::e#53\nFeb  4 23:00:58 debian named[20369]: network unreachable resolving 'com/DS/IN': 2001:500:2f::f#53\nFeb  4 23:00:58 debian named[20369]: network unreachable resolving 'com/DS/IN': 2001:7fd::1#53\nFeb  4 23:00:58 debian named[20369]: network unreachable resolving 'com/DS/IN': 2001:503:ba3e::2:30#53\nFeb  4 23:00:58 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 227.\nFeb  4 23:00:58 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:58 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:00:58 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:00:59 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:00:59] [Logs] /var/log/dmesg not found. trying to capture from cli\nFeb  4 23:00:59 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:59] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:00:59 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:00:59] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710064", "Location":"/var/log/syslog", "Content":"Feb  4 23:01:02 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:01:02] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nFeb  4 23:01:02 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:01:02] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nFeb  4 23:01:02 debian systemd[1]: Stopping BIND Domain Name Server...\nFeb  4 23:01:02 debian named[20369]: received control channel command 'stop'\nFeb  4 23:01:02 debian named[20369]: no longer listening on 127.0.0.1#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on 172.30.0.13#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on 10.1.1.3#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on 10.1.2.2#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on 10.0.3.2#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on ::1#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on fe80::ac66:e8ff:fe0f:8dfc%2#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on fe80::5c83:65ff:fefd:d945%3#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on fe80::9878:c9ff:fe27:fbb0%4#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on fe80::502a:9bff:fe92:3233%5#53\nFeb  4 23:01:02 debian named[20369]: shutting down: flushing changes\nFeb  4 23:01:02 debian named[20369]: stopping command channel on 127.0.0.1#953\nFeb  4 23:01:02 debian named[20369]: stopping command channel on ::1#953\nFeb  4 23:01:02 debian named[20369]: exiting\nFeb  4 23:01:02 debian systemd[1]: named.service: Succeeded.\nFeb  4 23:01:02 debian systemd[1]: Stopped BIND Domain Name Server.\nFeb  4 23:01:02 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:01:02] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nFeb  4 23:01:02 debian systemd[1]: session-133.scope: Succeeded.\nFeb  4 23:01:03 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:01:03] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:04 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:04] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710064", "Location":"/var/log/auth.log", "Content":"Feb  4 23:01:02 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:02 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nFeb  4 23:01:02 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:01:02 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:02 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nFeb  4 23:01:02 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:01:02 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:02 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nFeb  4 23:01:02 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:01:02 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:02 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.109.trace\nFeb  4 23:01:02 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:01:02 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:02 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.109.trace\nFeb  4 23:01:02 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:01:02 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:02 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/systemctl stop named\nFeb  4 23:01:02 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nFeb  4 23:01:02 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:02 debian sshd[20509]: Received disconnect from 192.168.254.2 port 44338:11: disconnected by user\nFeb  4 23:01:02 debian sshd[20509]: Disconnected from user blankcanvas 192.168.254.2 port 44338\nFeb  4 23:01:02 debian sshd[20484]: pam_unix(sshd:session): session closed for user blankcanvas\nFeb  4 23:01:02 debian systemd-logind[390]: Session 133 logged out. Waiting for processes to exit.\nFeb  4 23:01:02 debian systemd-logind[390]: Removed session 133.\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710064", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:01:02 debian systemd[1]: Stopping BIND Domain Name Server...\nFeb  4 23:01:02 debian named[20369]: received control channel command 'stop'\nFeb  4 23:01:02 debian named[20369]: no longer listening on 127.0.0.1#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on 172.30.0.13#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on 10.1.1.3#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on 10.1.2.2#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on 10.0.3.2#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on ::1#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on fe80::ac66:e8ff:fe0f:8dfc%2#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on fe80::5c83:65ff:fefd:d945%3#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on fe80::9878:c9ff:fe27:fbb0%4#53\nFeb  4 23:01:02 debian named[20369]: no longer listening on fe80::502a:9bff:fe92:3233%5#53\nFeb  4 23:01:02 debian named[20369]: shutting down: flushing changes\nFeb  4 23:01:02 debian named[20369]: stopping command channel on 127.0.0.1#953\nFeb  4 23:01:02 debian named[20369]: stopping command channel on ::1#953\nFeb  4 23:01:02 debian named[20369]: exiting\nFeb  4 23:01:02 debian systemd[1]: named.service: Succeeded.\nFeb  4 23:01:02 debian systemd[1]: Stopped BIND Domain Name Server.\nFeb  4 23:01:02 debian discern-file-sorcerer[17188]: #033[31mERRO#033[0m[23:01:02] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nFeb  4 23:01:02 debian systemd[1]: session-133.scope: Succeeded.\nFeb  4 23:01:03 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:01:03] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:04 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:04] [Logs] /var/log/dmesg not found. trying to capture from cli\nFeb  4 23:01:04 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:01:04] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:04 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:01:04] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710069", "Location":"/var/log/syslog", "Content":"Feb  4 23:01:04 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:01:04] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:04 debian discern-logs-sorcerer[17232]: #033[31mERRO#033[0m[23:01:04] [Logs] error saving /var/log/daemon.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:04 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:04] [Logs] Sending has completed\nFeb  4 23:01:04 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:01:04] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:04 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:04] [Proc] stripping info for: 20491\nFeb  4 23:01:04 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:04] [Proc] stripping info for: 20492\nFeb  4 23:01:04 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 229.\nFeb  4 23:01:04 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:04 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:04 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:06 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:06] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:01:07 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:01:07] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:08 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 230.\nFeb  4 23:01:08 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:08 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:08 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:09 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:09] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710069", "Location":"/var/log/auth.log", "Content":"Feb  4 23:01:08 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:08 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execve.bt\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:08 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat.bt\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:08 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710069", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:01:04 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:04] [Logs] Sending has completed\nFeb  4 23:01:04 debian discern-network-sorcerer[17257]: #033[31mERRO#033[0m[23:01:04] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:04 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:04] [Proc] stripping info for: 20491\nFeb  4 23:01:04 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:04] [Proc] stripping info for: 20492\nFeb  4 23:01:04 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 229.\nFeb  4 23:01:04 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:04 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:04 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:06 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:06] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:01:07 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:01:07] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:08 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 230.\nFeb  4 23:01:08 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:08 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:08 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:09 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:09] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710074", "Location":"/var/log/syslog", "Content":"Feb  4 23:01:12 debian systemd[20491]: gpg-agent-ssh.socket: Succeeded.\nFeb  4 23:01:12 debian systemd[20491]: Closed GnuPG cryptographic agent (ssh-agent emulation).\nFeb  4 23:01:12 debian systemd[20491]: gpg-agent.socket: Succeeded.\nFeb  4 23:01:12 debian systemd[20491]: Closed GnuPG cryptographic agent and passphrase cache.\nFeb  4 23:01:12 debian systemd[20491]: Removed slice User Application Slice.\nFeb  4 23:01:12 debian systemd[20491]: Reached target Shutdown.\nFeb  4 23:01:12 debian systemd[20491]: systemd-exit.service: Succeeded.\nFeb  4 23:01:12 debian systemd[20491]: Finished Exit the Session.\nFeb  4 23:01:12 debian systemd[20491]: Reached target Exit the Session.\nFeb  4 23:01:12 debian systemd[1]: user@1001.service: Succeeded.\nFeb  4 23:01:12 debian systemd[1]: Stopped User Manager for UID 1001.\nFeb  4 23:01:12 debian systemd[1]: Stopping User Runtime Directory /run/user/1001...\nFeb  4 23:01:12 debian systemd[1]: run-user-1001.mount: Succeeded.\nFeb  4 23:01:12 debian systemd[1]: user-runtime-dir@1001.service: Succeeded.\nFeb  4 23:01:12 debian systemd[1]: Stopped User Runtime Directory /run/user/1001.\nFeb  4 23:01:12 debian systemd[1]: Removed slice User Slice of UID 1001.\nFeb  4 23:01:13 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:01:13] [Ansible] saving config specified in env\nFeb  4 23:01:13 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:01:13] [Ansible] saving config and playbooks from home\nFeb  4 23:01:13 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:01:13] [Ansible] saving config from ~ and /etc\nFeb  4 23:01:13 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:01:13] [Ansible] finished reading. looping\nFeb  4 23:01:13 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:01:13] [Ansible] Saving playbooks from /etc/ansible\nFeb  4 23:01:13 debian discern-ansible-sorcerer[17057]: #033[31mERRO#033[0m[23:01:13] [Ansible] error reading /etc/ansible: stat /etc/ansible: no such file or directory\nFeb  4 23:01:13 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:01:13] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:14 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:14] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710074", "Location":"/var/log/auth.log", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710074", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:01:12 debian systemd[20491]: gpg-agent-ssh.socket: Succeeded.\nFeb  4 23:01:12 debian systemd[20491]: Closed GnuPG cryptographic agent (ssh-agent emulation).\nFeb  4 23:01:12 debian systemd[20491]: gpg-agent.socket: Succeeded.\nFeb  4 23:01:12 debian systemd[20491]: Closed GnuPG cryptographic agent and passphrase cache.\nFeb  4 23:01:12 debian systemd[20491]: Removed slice User Application Slice.\nFeb  4 23:01:12 debian systemd[20491]: Reached target Shutdown.\nFeb  4 23:01:12 debian systemd[20491]: systemd-exit.service: Succeeded.\nFeb  4 23:01:12 debian systemd[20491]: Finished Exit the Session.\nFeb  4 23:01:12 debian systemd[20491]: Reached target Exit the Session.\nFeb  4 23:01:12 debian systemd[1]: user@1001.service: Succeeded.\nFeb  4 23:01:12 debian systemd[1]: Stopped User Manager for UID 1001.\nFeb  4 23:01:12 debian systemd[1]: Stopping User Runtime Directory /run/user/1001...\nFeb  4 23:01:12 debian systemd[1]: run-user-1001.mount: Succeeded.\nFeb  4 23:01:12 debian systemd[1]: user-runtime-dir@1001.service: Succeeded.\nFeb  4 23:01:12 debian systemd[1]: Stopped User Runtime Directory /run/user/1001.\nFeb  4 23:01:12 debian systemd[1]: Removed slice User Slice of UID 1001.\nFeb  4 23:01:13 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:01:13] [Ansible] saving config specified in env\nFeb  4 23:01:13 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:01:13] [Ansible] saving config and playbooks from home\nFeb  4 23:01:13 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:01:13] [Ansible] saving config from ~ and /etc\nFeb  4 23:01:13 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:01:13] [Ansible] finished reading. looping\nFeb  4 23:01:13 debian discern-ansible-sorcerer[17057]: #033[36mINFO#033[0m[23:01:13] [Ansible] Saving playbooks from /etc/ansible\nFeb  4 23:01:13 debian discern-ansible-sorcerer[17057]: #033[31mERRO#033[0m[23:01:13] [Ansible] error reading /etc/ansible: stat /etc/ansible: no such file or directory\nFeb  4 23:01:13 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:01:13] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:14 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:14] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710079", "Location":"/var/log/syslog", "Content":"Feb  4 23:01:14 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 232.\nFeb  4 23:01:14 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:14 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:14 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:14 debian discern-jupyter-sorcerer[17211]: #033[36mINFO#033[0m[23:01:14] [Jupyter] File search & saving stopped\nFeb  4 23:01:16 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:16] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:01:17 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 233.\nFeb  4 23:01:17 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:17 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:17 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:19 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:19] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710079", "Location":"/var/log/auth.log", "Content":"Feb  4 23:01:14 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/sendto.bt\nFeb  4 23:01:14 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/sendmsg.bt\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:14 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710079", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:01:14 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 232.\nFeb  4 23:01:14 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:14 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:14 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:14 debian discern-jupyter-sorcerer[17211]: #033[36mINFO#033[0m[23:01:14] [Jupyter] File search & saving stopped\nFeb  4 23:01:16 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:16] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:01:17 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 233.\nFeb  4 23:01:17 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:17 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:17 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:19 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:19] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710084", "Location":"/var/log/syslog", "Content":"Feb  4 23:01:19 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:01:19] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:21 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 234.\nFeb  4 23:01:21 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:21 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:21 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:21 debian discern-network-sorcerer[17257]: #033[36mINFO#033[0m[23:01:21] [Network] saved network slice\nFeb  4 23:01:21 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:21] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:01:24 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 235.\nFeb  4 23:01:24 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:24 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:24 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:24 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:24] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710084", "Location":"/var/log/auth.log", "Content":"Feb  4 23:01:21 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvmmsg.bt\nFeb  4 23:01:21 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execveat.bt\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:21 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/closerange.bt\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:21 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:21 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710084", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:01:19 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:01:19] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:21 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 234.\nFeb  4 23:01:21 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:21 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:21 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:21 debian discern-network-sorcerer[17257]: #033[36mINFO#033[0m[23:01:21] [Network] saved network slice\nFeb  4 23:01:21 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:21] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:01:24 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 235.\nFeb  4 23:01:24 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:24 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:24 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:24 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:24] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710089", "Location":"/var/log/syslog", "Content":"Feb  4 23:01:25 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:01:25] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:26 debian discern-network-sorcerer[17257]: #033[36mINFO#033[0m[23:01:26] [Network] saved network slice\nFeb  4 23:01:26 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:26] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:01:27 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 236.\nFeb  4 23:01:27 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:27 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:27 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:29 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:29] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710089", "Location":"/var/log/auth.log", "Content":"Feb  4 23:01:27 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvfrom.bt\nFeb  4 23:01:27 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvmmsg.bt\nFeb  4 23:01:27 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execveat.bt\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:27 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execve.bt\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:27 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat.bt\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:27 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:27 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710089", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:01:25 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:01:25] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:26 debian discern-network-sorcerer[17257]: #033[36mINFO#033[0m[23:01:26] [Network] saved network slice\nFeb  4 23:01:26 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:26] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:01:27 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 236.\nFeb  4 23:01:27 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:27 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:27 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:29 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:29] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710094", "Location":"/var/log/syslog", "Content":"Feb  4 23:01:30 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 237.\nFeb  4 23:01:30 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:30 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:30 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:31 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:01:31] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:31 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:31] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:01:34 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 238.\nFeb  4 23:01:34 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:34 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:34 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:34 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:34] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710094", "Location":"/var/log/auth.log", "Content":"Feb  4 23:01:34 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvfrom.bt\nFeb  4 23:01:34 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvmmsg.bt\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:34 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat2.bt\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:34 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat.bt\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:34 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:34 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session closed for user root\nFeb  4 23:01:34 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"1738710094", "Location":"/var/log/daemon.log", "Content":"Feb  4 23:01:30 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 237.\nFeb  4 23:01:30 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:30 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:30 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:31 debian discern-cpu-sorcerer[2949]: #033[31mERRO#033[0m[23:01:31] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nFeb  4 23:01:31 debian discern-proc-sorcerer[20397]: #033[36mINFO#033[0m[23:01:31] [Proc] successfully saved data to the fusion core in normal loop\nFeb  4 23:01:34 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 238.\nFeb  4 23:01:34 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:34 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nFeb  4 23:01:34 debian systemd[1]: discern-os-record.service: Succeeded.\nFeb  4 23:01:34 debian discern-logs-sorcerer[17232]: #033[36mINFO#033[0m[23:01:34] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
